c2rust-refactor/src/scripting.rs

```diff
@@ -495,17 +750,13 @@ impl<'a, 'tcx> TransformCtxt<'a, 'tcx> {
         lua_ctx: LuaContext<'lua>,
         node: LuaAstNode,
     ) -> LuaResult<LuaTable<'lua>> {
-        let mut ast = lua_ctx.create_table()?;
         match self.clone_ast(node) {
-            RustAstNode::Stmt(s) => s.into_lua_ast(self, &mut ast)?,
-            RustAstNode::Expr(e) => e.into_lua_ast(self, &mut ast)?,
-            _ => {
-                return Err(LuaError::external(
-                    "get_ast not implemented for this type of RustAstNode",
-                ));
-            }
+            RustAstNode::Stmt(s) => s.into_lua_ast(self, lua_ctx),
+            RustAstNode::Expr(e) => e.into_lua_ast(self, lua_ctx),
```

Comment:
You could match both at the same time using `RustAstNode::Stmt(e) | RustAstNode::Expr(e)`.

Reply:
Are you thinking of `Semi`? `Stmt` won't work with `Expr`

Reply:
Oh right, I was thinking of `Semi`.